spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/interpreter/rules/mapping/MappingInterpreterSuite.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.conformance.interpreter.rules.mapping
+
+import org.apache.commons.io.IOUtils
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.funsuite.AnyFunSuite
+import za.co.absa.enceladus.conformance.interpreter.rules.testcasefactories.{NestedTestCaseFactory, SimpleTestCaseFactory}
+import za.co.absa.enceladus.utils.testUtils.{HadoopFsTestBase, LoggerTestBase, SparkTestBase}
+
+trait MappingInterpreterSuite extends AnyFunSuite with SparkTestBase with LoggerTestBase with BeforeAndAfterAll with HadoopFsTestBase{
+
+  protected val simpleTestCaseFactory = new SimpleTestCaseFactory()
+  protected val nestedTestCaseFactory = new NestedTestCaseFactory()
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    simpleTestCaseFactory.createMappingTables()
+    nestedTestCaseFactory.createMappingTables()
+  }
+
+  override def afterAll(): Unit = {
+    simpleTestCaseFactory.deleteMappingTables()
+    nestedTestCaseFactory.deleteMappingTables()
+    super.afterAll()
+  }
+
+  protected def cleanupContainsNullProperty(inputSchemaTree: String): String = {
```

Comment:
I can see that this is was in the existing code, too, but I am missing the information why it is ok to disregard this difference. Naively, this seems like a possible place to miss a schema difference. Maybe brief information on this could be present here or at the place of usage?

Reply:
@benedeki ?